### PR TITLE
UnixPB: Update CRIU role for Ubuntu

### PR DESCRIPTION
**Pull request by vsebe**
Upgrade version from v3.16 to 3.17.
Add cap_checkpoint_restore capabilities on Ubuntu 22.

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

##### Checklist

- [ ] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Review by sxa — APPROVED** [review]
Not run the adoptium infrastructure, so I'm ok with this.

**Comment by sxa** [comment]
> Friendly bump on this, or are we waiting till post release at this point?

I wasn't, but it's awaiting a second reviewer before I can merge. @karianna are you willing to do the honours? And are you ok with merging during the lockdown cycle given that it will not affect the adoptium machines.

**Comment by gdams** [comment]
> And are you ok with merging during the lockdown cycle given that it will not affect the adoptium machines.

Let's try an approval process :)

**Review by gdams — APPROVED** [review]

**Pull request merged**
